src/LibraryManager.Vsix/UI/Extensions/RemoveSubstringExtension.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Windows.Markup;
+
+namespace Microsoft.Web.LibraryManager.Vsix.UI.Extensions
+{
+    [MarkupExtensionReturnType(typeof(string))]
+    internal class RemoveSubstringExtension : MarkupExtension
+    {
+        private object _text;
+        private string _remove;
+
+        public RemoveSubstringExtension(object text)
+            : this(text, null)
+        {
+        }
+
+        public RemoveSubstringExtension(object text, string remove)
+        {
+            _text = text;
+            _remove = remove;
+        }
+
+        public override object ProvideValue(IServiceProvider serviceProvider)
+        {
+            if (_text == null)
+            {
+                return null;
+            }
+
+            string remove = _remove ?? "_";
```

Comment:
I would do this in the constructor instead of here.

Reply:
Updated here:
https://github.com/aspnet/LibraryManager/commit/fb24f8124cac6e0f58dd12ded43b8c1b99029fbb